Fixes the bindings generator to exclude phonon and xml-patterns, since those are
excluded from the recommended build of Qt for KDE development.

diff -ur qtscriptgenerator-git/generator/generator.qrc qtscriptgenerator-git-build/generator/generator.qrc
--- qtscriptgenerator-git/generator/generator.qrc	2009-05-16 14:09:48.000000000 +0100
+++ qtscriptgenerator-git-build/generator/generator.qrc	2009-05-16 14:37:14.000000000 +0100
@@ -9,9 +9,7 @@
 <file alias="typesystem_svg.txt">typesystem_svg.xml</file>
 <file alias="typesystem_network.txt">typesystem_network.xml</file>
 <file alias="typesystem_xml.txt">typesystem_xml.xml</file>
-<file alias="typesystem_phonon.txt">typesystem_phonon.xml</file>
 <file alias="typesystem_webkit.txt">typesystem_webkit.xml</file>
-<file alias="typesystem_xmlpatterns.txt">typesystem_xmlpatterns.xml</file>
 <file>parser/rpp/pp-qt-configuration</file>
 </qresource>
 </RCC>
diff -ur qtscriptgenerator-git/qtbindings/qtbindings.pro qtscriptgenerator-git-build/qtbindings/qtbindings.pro
--- qtscriptgenerator-git/qtbindings/qtbindings.pro	2009-05-16 14:09:48.000000000 +0100
+++ qtscriptgenerator-git-build/qtbindings/qtbindings.pro	2009-05-16 14:37:29.000000000 +0100
@@ -6,8 +6,6 @@
 	  qtscript_sql \
 	  qtscript_svg \
 	  qtscript_xml \
-	  qtscript_phonon \
 	  qtscript_webkit \
-	  qtscript_xmlpatterns \
 	  qtscript_uitools \
 	  qs_eval
